frontend/src/components/entity/EntityImagery.tsx

```diff
@@ -83,25 +79,20 @@ const EntityImagery = ({
   compact?: boolean;
 }) => {
   if (entity.type === TypeEnum.VIDEO) {
-    const player = (
-      <VideoPlayer
-        videoId={entity.metadata.video_id}
-        duration={entity.metadata.duration}
-      />
+    return (
+      <Box
+        sx={{
+          aspectRatio: '16 / 9',
+          width: '100%',
+          ...(compact ? {} : { minWidth: '240px' }),
```

Comment:
One side-effect is that the video width will change on larger screens when the video is being played:
![image](https://user-images.githubusercontent.com/4726554/168540581-bc742dc4-9b29-4ec2-b570-b88e03b75008.png)


A potential fix:
```suggestion
          ...(compact ? {} : { minWidth: '240px', maxWidth: { sm: '240px' } }),
```